types/linkifyjs/linkifyjs-tests.ts

```diff
@@ -1,141 +1,163 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
 import linkifyHtml from "linkifyjs/html";
+import Linkify from "linkifyjs/react";
+
+declare function describe(desc: string, f: () => void): void;
+declare function it(desc: string, f: () => void): void;
 
 // From the docs here: https://soapbox.github.io/linkifyjs/docs/options.html
```

Comment:
Since this comment doesn't apply to the section you added, can you move it to just under the `describe("linkifyjs/html")` line?